doc/Administration-guide/System-Configuration/General-configuration/index.md

```diff
@@ -4,4 +4,183 @@ title: General configuration
 permalink: /:path/
 sort_idx: 10
 ---
-Kaa node configuration. Inter-node communication and networking config.
+
+After Kaa installation on Ubuntu/Debian OS (deb packages), configuration files for each Kaa component will be extracted into the
+/usr/lib/kaa-node/conf or /etc/kaa-node/conf directories.
+
+The kaa-node.properties file is responsible for Kaa server configuration.
+
+>**NOTE:**
+> After changing properties in the kaa-node.properties file, you must restart the node for changes to take effect, by executing following command:
+>
+```bash
+$ sudo service kaa-node restart
+```
+
+The kaa-node.properties consist of the following parameters:
+
+#### control_service_enabled
+Default: _true_
+
+Specifies if Control Service is enabled.
+
+#### bootstrap_service_enabled
+Default: _true_
+
+Specifies if Bootstrap Service is enabled.
+
+#### operations_service_enabled
+Default: _true_
+
+Specifies if Operations Service is enabled.
+
+#### thrift_host
+Default: _localhost_
+
+Thrift service host address.
+
+#### thrift_port
+Default: _9090_
+
+Thrift service port address.
```

Comment:
This information is used for Thrift remote procedure calls between nodes in Kaa cluster.